src/components/Alteration/AlterationSummary.vue

```diff
@@ -73,10 +73,7 @@
     <template v-if="haveNameTranslationsChanged">
       <v-divider class="mx-4" />
       <div class="section-container name-translation-summary">
-        <NameTranslation
-          :nameTranslations="getNameTranslations"
```

Comment:
Prop no longer needed -- the NameTranslation component gets the store value directly.